docs/compile.md

```diff
@@ -188,7 +188,69 @@ It's up to you to decide what the output is.
 
 ### kadet
 
-This input type is experimental. See <https://github.com/deepmind/kapitan/pull/190> for its usage.
+Kadet is an extensible input type that enables you to generate templates using python. The key benefit being the ability to utilize familiar programing principles while having access to kapitan's powerful inventory system. 
+
+A library that defines resources as classes using the Base Object class (BaseObj) is required. These can then be utilized within components to render output. 
+
+The following functions are provided by the class BaseObj.
+
+Method definitions:<br/>
```

Comment:
You don't need these <br/> breakpoints  in markdown, just leave an empty newline instead and it will render it  correctly.
Same for all  these breaklines below.

Reply:
Sorry for making you review these simple mistakes. Initially, it did not render as intended so I looked for a workaround. I added the newline's as recommended and it seems fine.